lib/client/js/ui/controls/Header.js

```diff
@@ -32,8 +35,13 @@ var Header = (function () {
             .append(
                 $("<h1>", { "class": "beans--Header-title" })
                     .text("Beans")
-            );
+            )
+            .append(this._AuthButton.view());
```

Comment:
`append` takes in any number of arguments, so instead of calling it again we can just pass the `AuthButton`'s view as a second argument to the first call to `append`.